2018-03-15 11:14:08,864 INFO generated testing report: /home/ubuntu/study/selenium2/reports/2018-03-15_11_14_08_result.html [log.py:47]
2018-03-15 11:14:08,981 INFO read csv file [log.py:47]
2018-03-15 11:14:08,993 INFO username list: ['doris_test@sina.com', 'doris_test@sina.com', '', 'doris_test@sina.com'] [log.py:47]
2018-03-15 11:14:08,994 INFO password list: ['admin@123', '', 'admin@123', '123456'] [log.py:47]
2018-03-15 11:14:44,396 INFO username: doris_test@sina.com, password:  [log.py:47]
2018-03-15 11:14:54,510 INFO username: , password: admin@123 [log.py:47]
2018-03-15 11:15:03,897 INFO username:doris_test@sina.com, password: admin@123 [log.py:47]
2018-03-15 11:17:38,818 INFO username: doris_test@sina.com, password: 123456 [log.py:47]
2018-03-15 11:17:44,627 INFO none of wrong images [log.py:47]
2018-03-15 11:17:44,630 INFO flag is none [log.py:47]
2018-03-15 11:17:44,631 INFO the newest report is 2018-03-15_11_14_08_result.html [log.py:47]
2018-03-15 11:17:44,632 INFO preparing send report /home/ubuntu/study/selenium2/reports/2018-03-15_11_14_08_result.html [log.py:47]
2018-03-15 11:17:44,649 INFO attach reports [log.py:47]
2018-03-15 11:18:15,149 ERROR send email failed [log.py:53]
2018-03-15 14:12:06,881 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:12:33,465 INFO generated testing report: /home/ubuntu/study/selenium2/reports/2018-03-15_14-12-33_result.html [log.py:47]
2018-03-15 14:12:33,485 INFO read csv file [log.py:47]
2018-03-15 14:12:33,497 INFO username list: ['doris_test@sina.com', 'doris_test@sina.com', '', 'doris_test@sina.com'] [log.py:47]
2018-03-15 14:12:33,497 INFO password list: ['admin@123', '', 'admin@123', '123456'] [log.py:47]
2018-03-15 14:12:33,498 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:12:38,334 INFO username: doris_test@sina.com, password:  [log.py:47]
2018-03-15 14:12:42,884 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:12:49,409 INFO username: , password: admin@123 [log.py:47]
2018-03-15 14:12:52,934 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:12:57,792 INFO username:doris_test@sina.com, password: admin@123 [log.py:47]
2018-03-15 14:13:23,596 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:13:28,287 INFO username: doris_test@sina.com, password: 123456 [log.py:47]
2018-03-15 14:13:33,865 INFO none of wrong images [log.py:47]
2018-03-15 14:13:33,866 INFO flag is none [log.py:47]
2018-03-15 14:13:33,866 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:13:35,743 INFO the newest report is 2018-03-15_14-12-33_result.html [log.py:47]
2018-03-15 14:13:35,743 INFO preparing send report /home/ubuntu/study/selenium2/reports/2018-03-15_14-12-33_result.html [log.py:47]
2018-03-15 14:13:35,746 INFO attach reports [log.py:47]
2018-03-15 14:14:06,266 ERROR send email failed [log.py:53]
2018-03-15 14:57:37,418 INFO generated testing report: /home/ubuntu/study/selenium2/reports/2018-03-15_14-57-37_result.html [log.py:47]
2018-03-15 14:57:37,422 INFO read csv file [log.py:47]
2018-03-15 14:57:37,423 INFO username list: ['doris_test@sina.com', 'doris_test@sina.com', '', 'doris_test@sina.com'] [log.py:47]
2018-03-15 14:57:37,423 INFO password list: ['admin@123', '', 'admin@123', '123456'] [log.py:47]
2018-03-15 14:57:37,424 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:57:42,731 INFO username: doris_test@sina.com, password:  [log.py:47]
2018-03-15 14:57:47,269 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:57:52,222 INFO username: , password: admin@123 [log.py:47]
2018-03-15 14:57:55,670 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:58:00,255 INFO username:doris_test@sina.com, password: admin@123 [log.py:47]
2018-03-15 14:58:26,016 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:58:31,295 INFO username: doris_test@sina.com, password: 123456 [log.py:47]
2018-03-15 14:58:36,802 INFO none of wrong images [log.py:47]
2018-03-15 14:58:36,803 INFO flag is none [log.py:47]
2018-03-15 14:58:36,803 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 14:58:38,663 INFO the newest report is 2018-03-15_14-57-37_result.html [log.py:47]
2018-03-15 14:58:38,664 INFO preparing send report /home/ubuntu/study/selenium2/reports/2018-03-15_14-57-37_result.html [log.py:47]
2018-03-15 14:58:38,666 INFO attach reports [log.py:47]
2018-03-15 14:59:09,268 ERROR send email failed [log.py:53]
2018-03-15 15:47:54,309 INFO generated testing report: /home/ubuntu/study/selenium2/reports/2018-03-15_15-47-54_result.html [log.py:47]
2018-03-15 15:47:54,385 INFO read csv file [log.py:47]
2018-03-15 15:47:54,386 INFO username list: ['doris_test@sina.com', 'doris_test@sina.com', '', 'doris_test@sina.com'] [log.py:47]
2018-03-15 15:47:54,386 INFO password list: ['admin@123', '', 'admin@123', '123456'] [log.py:47]
2018-03-15 15:47:54,387 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 15:48:00,068 INFO username: doris_test@sina.com, password:  [log.py:47]
2018-03-15 15:48:04,600 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 15:48:10,635 INFO username: , password: admin@123 [log.py:47]
2018-03-15 15:48:14,081 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 15:48:19,081 INFO username:doris_test@sina.com, password: admin@123 [log.py:47]
2018-03-15 15:48:44,919 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 15:48:49,606 INFO username: doris_test@sina.com, password: 123456 [log.py:47]
2018-03-15 15:48:55,173 INFO none of wrong images [log.py:47]
2018-03-15 15:48:55,174 INFO flag is none [log.py:47]
2018-03-15 15:48:55,175 INFO host:http://172.17.1.207:5555/wd/hub,browser:chrome [log.py:47]
2018-03-15 15:48:57,052 INFO the newest report is 2018-03-15_15-47-54_result.html [log.py:47]
2018-03-15 15:48:57,053 INFO preparing send report /home/ubuntu/study/selenium2/reports/2018-03-15_15-47-54_result.html [log.py:47]
2018-03-15 15:48:57,055 INFO attach reports [log.py:47]
2018-03-15 15:49:27,567 ERROR send email failed [log.py:53]
